test/Dialect/FIRRTL/annotations.mlir

```diff
@@ -1905,3 +1905,19 @@ firrtl.circuit "Top" attributes {
     firrtl.instance dut @DUT()
   }
 }
+
+// -----
+
+// CHECK-LABEL: firrtl.circuit "Top"
+firrtl.circuit "Top" attributes {
+  rawAnnotations = [
+    {
+      class = "circt.OutputDirAnnotation",
+      dirname = "foobarbaz",
```

Comment:
Two more cases may be interesting to test:

1. `foobarbaz/`
2. `foobarbaz/qux`